website/layouts/_shortcodes/tab-group.html

```
@@ -0,0 +1,12 @@
+{{- $id := .Get 0 -}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)